pipeline/scripts/AddOnFunctions/check_same_samplename.R

```diff
@@ -0,0 +1,4 @@
+# function to test whether intensity and Z-score columns match
+same_samplename <- function(int_col_name, zscore_col_name) {
```

Comment:
The function name does not correspond with the changed filename, might therefore not be called (if it's still being used at all).

Reply:
Function name changed to check_same_samplename